### PR TITLE
Submit Assignment-1

### PR DESCRIPTION
**Pull request by deysoumyadey**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Complete Lab1 and Lab2.
Complete assignment 1.

## What did you learn from the changes you have made?
Learnt basic concepts of deep learning using Keras. Created and trained a CNN model along with some enhancements of the model using batch normalization and maxpooling.

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
Would like to explore more options for training a CNN model.

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
Finding the right balance of hyperparameters is a challenging task. Used some LLMs and outside resources to learn more about the hyperparameters of Keras.

## How were these changes tested?
N/A

## Checklist
- [x] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by DwardEE — APPROVED** [review]
Good job! Your baseline accuracy is quite a bit lower than expected, and your final accuracy is still just a shy under what is optimal. So there are some improvements that be made to the model. I'll mark at the assignment as complete, but feel free to play around with your model to see if you can slightly higher accuracies.
